README.md

```diff
@@ -34,6 +34,16 @@ module.exports = {
       },
       requiredConfig: ['awesomeApiKey'], // throw an error if this is not configured
 
+      // return an object from here to add functionality to the `pluginHelper`
+      // that is passed in to user defined config property functions
+      pluginHelper: function(context) {
+        return {
+          sayHello: function() {
+            return 'Hello' + context.project.name();
+          }
```

Comment:
Yep. Was low on inspiration this evening. Did you have any thoughts on how people might use this but of functionality? Can't really think of how I might use this in the real world to be honest.